branch_checkout.go

```diff
@@ -21,6 +21,7 @@ type checkoutOptions struct {
 type branchCheckoutCmd struct {
 	checkoutOptions
 
+	Sort      string `short:"s" optional:"" config:"branchCheckout.sort" help:"Sort branches by the given field"`
```

Comment:
minor: I'd like to keep the short form '-s' free as there's a good chance something that's used more frequently than sort on the CLI will want it. (Also, optional is the default for flags so we can drop it.)
```suggestion
	Sort      string `config:"branchCheckout.sort" help:"Sort branches by the given field"`
```